react-app/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "react-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "npx webpack --config webpack/webpack.dev.js",
+    "develop": "webpack -d --watch --config ./webpack/webpack.dev.js --display-error-details",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Just a quick reminder - one of acceptance criteria for the module - you should have configured `test`-script to do something more, than simply throwing a message.